lib/ez_gen/file_writer.rb

```diff
@@ -0,0 +1,165 @@
+class FileWriter
+  def initialize(args)
+    @args = args
+    @lower_name = args.last
+    @upper_name = @lower_name.split('_').map(&:capitalize).join
+  end
+
+  def write_rake
+    rake = File.open("Rakefile", "w")
+    rake.write(
+    'require "rake/testtask"
+
+    Rake::TestTask.new do |task|
+      task.pattern = "test/*_test.rb"
+    end
+
+    desc "Run single test file based on class name, ex: rake test_class person"
+    task :test_class do
+      @args.each { |arg| task arg.to_sym do ; end }
+      eval File.read("./test/#{@lower_name}_test.rb")
+    end
+
+    desc "Generate new class and test, ex: rake g person"
+    task :g do
+      @args.each { |arg| task arg.to_sym do ; end }
+      ruby "./.scripts/generate.rb #{@lower_name}"
```

Comment:
This is related to the same thing as above--I was interpolating the args passed to ez_gen into this rake file, rather than allowing different args passed into the rake file after ez_gen creates the rake file.